core/src/tangram.cpp

```diff
@@ -75,9 +82,11 @@ class Map::Impl {
     TileWorker tileWorker{MAX_WORKERS};
     TileManager tileManager{tileWorker};
     MarkerManager markerManager;
+    FrameBuffer selectionBuffer{256, 256};
```

Comment:
Oh, yes. I've added a commit to visualize the actual selection framebuffer for testing